stable/nginx/nginx/templates/NetworkPolicy.yaml

```
@@ -0,0 +1,48 @@
+{{ if .Values.NetworkPolicy.Enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ template "nginx-fullname" . }}
+  labels:
+    app: {{ template "nginx.name" . }}
+    chart: {{ template "nginx.chart" . }}
+    release: {{ .Release.Name }}
+    instance: {{ .Values.Instance | quote }}
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ template "nginx.name" . }}
+      chart: {{ template "nginx.chart" . }}
+      release: {{ .Release.Name }}
+      instance: {{ .Values.Instance | quote }}
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress:
+  - from:
+    - ipBlock:
+        cidr: {{ .Values.NetworkPolicy.AllowedCIDR }}
+{{ end }}
+---
+{{ if not .Values.NetworkPolicy.Enabled }}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ template "nginx-fullname" . }}
+  labels:
+    app: {{ template "nginx.name" . }}
+    chart: {{ template "nginx.chart" . }}
+    release: {{ .Release.Name }}
+    instance: {{ .Values.Instance | quote }}
+spec:
+  podSelector:
+    matchLabels:
+      app: {{ template "nginx.name" . }}
+      chart: {{ template "nginx.chart" . }}
+      release: {{ .Release.Name }}
+      instance: {{ .Values.Instance | quote }}
+  policyTypes:
+  - Ingress
+  - Egress
+  ingress: {}
```

```suggestion
  egress:
  - {}
  ingress:
  - {}
```

Need to allow ingress and egress